<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourWidget</class>
 <widget class="QWidget" name="FourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FourWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="VideoWidget" name="widget_1" native="true"/>
   </item>
   <item>
    <widget class="VideoWidget" name="widget_2" native="true"/>
   </item>
   <item>
    <widget class="VideoWidget" name="widget_3" native="true"/>
   </item>
   <item>
    <widget class="VideoWidget" name="widget_4" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
